.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @cisagov/team-ois
+* @jsf9k
```

Comment:
Do you want to add someone else as a code owner here?  It's probably best if we have at least two owners for every repo.  I'm happy to volunteer.

Reply:
See commit 133f300.